distrho/extra/String.hpp

```diff
@@ -22,6 +22,11 @@
 
 #include <algorithm>
 
+#if defined(_MSC_VER)
```

Comment:
it is likely that this will be needed in a few other places.
As such it should go to DistrhoUtils.hpp